src/main/java/ch/bfh/bti7081/view/NewSeminarView.java

```diff
@@ -0,0 +1,157 @@
+package ch.bfh.bti7081.view;
+
+import ch.bfh.bti7081.model.manager.SeminarCategoryManager;
+import ch.bfh.bti7081.model.manager.SeminarManager;
+import ch.bfh.bti7081.model.seminar.Seminar;
+import ch.bfh.bti7081.model.seminar.SeminarCategory;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.select.Select;
+import com.vaadin.flow.component.textfield.NumberField;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.timepicker.TimePicker;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.BinderValidationStatus;
+import com.vaadin.flow.data.binder.BindingValidationStatus;
+import com.vaadin.flow.router.Route;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Route(value = "newSeminar", layout = Layout.class)
+public class NewSeminarView extends VerticalLayout {
+
+    private H1 title = new H1("Seminar erstellen");
+
+    private TextField seminarTitle = new TextField("Seminar-Titel");
+    private Select<SeminarCategory> seminarCategory = new Select<SeminarCategory>();
+
+    private TimePicker seminarTime = new TimePicker("Zeit");
+    private DatePicker seminarDate = new DatePicker("Datum");
+    private FormLayout dateComposite = new FormLayout(seminarDate,seminarTime);
+
+    private TextArea seminarDescription = new TextArea("Beschreibung");
+    private TextField seminarLink = new TextField("Externer Link");
+
+    private TextField seminarStreet = new TextField("Strasse");
+    private TextField seminarStreetNbr = new TextField("Nr.");
+    private FormLayout streetComposite = new FormLayout(seminarStreet,seminarStreetNbr);
+
+    private NumberField seminarPlz = new NumberField("PLZ");
+    private TextField seminarPlace = new TextField ("Ort");
+    private FormLayout placeComposite = new FormLayout(seminarPlz, seminarPlace);
+
+    private FormLayout seminarForm = new FormLayout();
+
+    private Label errorMessage = new Label("Hier könnte ihre Fehlermeldung stehen.");
+
+    private Button save = new Button("Save", new Icon(VaadinIcon.PLUS));
```

Comment:
mMn falsches Icon, zum speichern wird normalerweise kein + verwendet

Reply:
i ha ds no cool gfunge :)

Reply:
Habe mich auch informiert. Es gibt kein passendes Icon in der Vaadin-Library, das veraltete Disketten-Icon wollen sie nicht mehr verwenden, da es ein Relikt aus alten Zeiten ist.